core/parser.mly

```diff
@@ -1185,21 +1199,32 @@ vfield:
 | CONSTRUCTOR                                                  { ($1, present) }
 | CONSTRUCTOR fieldspec                                        { ($1, $2)      }
 
+tfields:
```

Comment:
The `fields_def` macro is supposed to eliminate some of the redundancy w.r.t. rows. We should use that rather than duplicating the parsing rules for rows.

Reply:
cf supra